### ISSUE TITLE
The sample WMS url which is used in the tutorial is no longer existed

### ISSUE THREAD

**Issue by gharibi** [Leaflet/Leaflet#5540]
In all of the samples and descriptions in http://leafletjs.com/examples/wms/wms.html regarding WMS, the sample WMS service which is http://demo.opengeo.org/geoserver/ows?service=wms&version=1.3.0&request=GetCapabilities is no longer existed or at least unavailable. Therefore it has made all of the examples, not visual.

Edit 2017-Jun-22: fix links

**Comment by IvanSanchez** [comment]
The WMS server is returning a HTTP code 503 AKA "Service Unavailable".

According to https://tools.ietf.org/html/rfc7231#section-6.6.4 :

> The 503 (Service Unavailable) status code indicates that the server is currently unable to handle the request due to a temporary overload or scheduled maintenance, which will likely be alleviated after some delay.

So I'll assume that the folks at Boundless are aware and will restart the WMS server in the near future. Please reopen if this doesn't happen in a few days.

(Also, if you know any other WMS server with worldwide coverage and better availability, you are welcome to edit the tutorials and change it)

**Comment by gharibi** [comment]
There is a stable server which I always use their services as test. I will modify the tutorial based on that service. However we need to close this issue when we resolve it.

**Comment by perliedman** [comment]
It appears this service has been 503 for at least three weeks now, so even though it's technically correct 503 is a temporary error, we should consider either contacting Boundless to resolve this, or switch to another service.

**Comment by astridx** [comment]
> (Also, if you know any other WMS server with worldwide coverage and better availability, you are welcome to edit the tutorials and change it)

Is it important, that it is a WMS server with worldwide coverage? For learning reasons I tried a server that coveraged only Germany: https://maps.dwd.de/geoserver/dwd/wms
This Server shows the weather and you can see if there ware warnings:

http://playground-leaflet.rhcloud.com/mexo/edit?html,output


In I book I found this server with an worldwide Satellit image:

```
var usgs = L.tileLayer.wms("http://basemap.nationalmap.gov/ArcGIS/services/USGSImageryTopo/MapServer/WMSServer", {
    layers:'0',
    format: 'image/png',
    transparent: true,
    attribution: "USGS"
}).addTo(map);
```

**Comment by jjimenezshaw** [comment]
Hi. I created PR #5712
I hope it works, but I don't know how to test the full 'compiled' html page.
Go to http://opengeo.org and you will understand what happened.

I think it is easier just updating the url. Changing to another service implies changing also the examples that refer to specific layers, etc. Too much work.

BTW, in `getCapabilities`, the field `OnlineResource` is worng :unamused:. If you use any software to process it, ignore that field (if you can).

**Comment by IvanSanchez** [comment]
Fixed by #5712
